stl/inc/algorithm

```diff
@@ -3567,6 +3567,77 @@ _FwdIt2 unique_copy(_ExPo&&, _FwdIt1 _First, _FwdIt1 _Last, _FwdIt2 _Dest) noexc
 }
 #endif // _HAS_CXX17
 
+#ifdef __cpp_lib_concepts
+namespace ranges {
+    // VARIABLE ranges::reverse
+    // clang-format off XXX

```

Comment:
Unintentional? (This is the only thing I'm Requesting Changes for)

Reply:
Derp. This, in combination with the commented-out _requires-clause_ below, is the "ask clang-format to double-check my formatting after substantially editing this function" pattern.